lib/api/response/survey_response.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'survey_response.g.dart';
+
+@JsonSerializable()
+class SurveyResponse {
+  String? id;
+  String? title;
```

Comment:
We will need to 

- switch the `background`

<img width="514" alt="image" src="https://user-images.githubusercontent.com/16315358/229668075-dfd78334-a496-4099-aa40-2604ba53f669.png">

- and show the `description`

Please check again the necessary fields for the following tasks.

Reply:
Aah yes, correct. 🙏 

I should have added them in this PR, please allow me to update these changes in the [integration PR](https://github.com/nimblehq/ic-flutter-taher-toby/pull/56) 🙇